push-build.sh

```diff
@@ -42,7 +42,8 @@ PROG=${0##*/}
 #+ OPTIONS
 #+     [--nomock]                - Enables a real push (--ci only)
 #+     [--federation]            - Enable FEDERATION push
-#+     [--ci]                    - Used when called from Jekins (for ci runs)
+#+     [--ci]                    - Used when called from Jenkins (for ci runs)
+#+     [--alias=]                - Used when need to upload to version alias txt (--ci only)
```

Comment:
I don't think `alias` is descriptive enough here.  How about `--extra-publish-file`.  Also the man page should describe the format of the arg which is a relative path that is appended to a GCS path.  That should be noted so the user understands how the arg is constructed and used.